weblab/datasets/tests/test_views.py

```diff
@@ -114,19 +114,22 @@ def test_add_delete_add_file(self, logged_in_user, client, helpers, public_proto
 
 
 @pytest.mark.django_db
-class TestEntityRenaming:
-    def test_dataset_renaming_success(self, client, logged_in_user, helpers):
+class TestDatasetRenaming:
+    def test_dataset_renaming_success(self, client, my_dataset_with_file):
 
-        dataset = recipes.dataset.make(author=logged_in_user)
-        assert dataset.name == 'my dataset1'
+        dataset = my_dataset_with_file
+        assert dataset.archive_path.exists()
```

Comment:
We could make this test slightly stronger with this change and the next below.
```suggestion
        old_path = dataset.archive_path
        assert old_path.exists()
```